jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,567 @@
+[
+	{
+		"name": "discv5_nodeInfo",
+		"summary": "Returns ENR and nodeId information of the local discv5 node.",
+		"params": [],
+		"result": {
+			"name": "nodeInfoResult",
+			"description": "Local node information",
+			"required": true,
+			"schema": {
+				"title": "nodeInfoResults",
+				"description": "ENR and NodeId of the local peer",
+				"type": "object",
+				"required": [
+					"enr",
+					"nodeId"
+				],
+				"properties": {
+					"enr": {
+						"title": "nodeENR",
+						"description": "URL-safe base64 encoded \\\"text\\\" version of the ENR prefixed by \\\"enr:\\\".\"",
+						"$ref": "#/components/schemas/Enr"
+					},
+					"nodeId": {
+						"title": "nodeId",
+						"description": "Hex encoded `NodeId` of an ENR (a 32 byte identifier).",
+						"$ref": "#/components/schemas/bytes32"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_updateNodeInfo",
+		"summary": "Add, update, or remove a key-value pair from the local node record",
+		"params": [
+			{
+				"name": "Key/value pairs",
+				"required": true,
+				"schema": {
+					"title": "",
+					"type": "array",
+					"items": { "$ref": "#/components/schemas/KVpairs" }
+			}
+			}
+		],
+		"result": {
+			"name": "nodeInfoResult",
+			"description": "Local node information",
+			"required": true,
+			"schema": {
+				"title": "nodeInfoResults",
+				"description": "ENR and NodeId of the local peer",
+				"type": "object",
+				"required": [
+					"enr",
+					"nodeId"
+				],
+				"properties": {
+					"enr": {
+						"title": "nodeENR",
+						"description": "URL-safe base64 encoded \\\"text\\\" version of the ENR prefixed by \\\"enr:\\\".\"",
+						"$ref": "#/components/schemas/Enr"
+					},
+					"nodeId": {
+						"title": "nodeId",
+						"description": "Hex encoded `NodeId` of an ENR (a 32 byte identifier).",
+						"$ref": "#/components/schemas/bytes32"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_routingTableInfo",
+		"summary": "Returns meta information about discv5 routing table.",
+		"params": [],
+		"result": {
+			"name": "routingTableInfoResult",
+			"description": "Discv5 routing table information",
+			"required": true,
+			"schema": {
+				"title": "routingTableInfoResults",
+				"description": "Routing table details",
+				"type": "object",
+				"required": [
+					"localKey",
+					"buckets"
+				],
+				"properties": {
+					"localKey": {
+						"title": "nodeId",
+						"description": "The key identifying the local peer that owns the routing table.",
+						"$ref": "#/components/schemas/bytes32"
+					},
+					"buckets":  {
+						"title": "kBucketsTable",
+						"description": "Represents a Kademlia routing table.",
+						"$ref": "#/components/schemas/kBuckets"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_setEnr",
```

Comment:
I think `discv5_addEnr` is a better name.

Also, database is mentioned in some of these ENR related calls, but shouldn't we call it routing table?